dsp-shared/src/main/scala/dsp/valueobjects/Iri.scala

```diff
@@ -221,6 +239,8 @@ object IriErrorMessages {
   val RoleIriInvalid     = (iri: String) => s"Role IRI: $iri is invalid."
   val UserIriMissing     = "User IRI cannot be empty."
   val UserIriInvalid     = (iri: String) => s"User IRI: $iri is invalid."
+  val UuidMissing        = "UUID cannot be empty"
+  val UuidInvalid        = (uuid: String) => s"'$uuid' is not a UUID"
   val UuidVersionInvalid = "Invalid UUID used to create IRI. Only versions 4 and 5 are supported."
```

Comment:
Partially true. Subject for value objects refactor, because it's also used in all IRIs as UUID validation. It should be extracted only to UUID value object which could be stored separately, but would trigger another refactor in this PR which wasn't a scope of it.